manifests/config.pp

```diff
@@ -4,6 +4,7 @@
     path    => $puppet::conf,
     ensure  => 'present',
     section => 'main',
+    notify  => Service['puppet_agent'],
```

Comment:
This one creates the failing tests, as Service['puppet_agent'] is defined after this class. I'll remove it for now, and I'll think about it better